_maps/shuttles/shiptest/syndicate_twinkleshine.dmm

```diff
@@ -1,7754 +1,9810 @@
-	},
-/obj/machinery/light,
-/obj/effect/turf_decal/corner/opaque/red/diagonal,
-/turf/open/floor/plasteel/white,
-/area/ship/medical)
-"zv" = (
+"wJ" = (
+/obj/effect/turf_decal/industrial/warning,
 /obj/structure/cable{
-	icon_state = "1-2"
-	},
-/obj/effect/turf_decal/industrial/warning{
-	dir = 8
+	icon_state = "4-8"
 	},
-/obj/machinery/door/firedoor/border_only{
+/obj/machinery/atmospherics/pipe/layer_manifold/visible{
 	dir = 8
 	},
+/obj/effect/decal/cleanable/dirt/dust,
 /turf/open/floor/engine,
-/area/ship/engineering)
-"zA" = (
-/obj/effect/turf_decal/industrial/warning{
-	dir = 4
+/area/ship/engineering/engine)
+"wL" = (
+/obj/effect/turf_decal/corner/opaque/syndiered{
+	dir = 10
 	},
-/obj/machinery/door/firedoor/border_only{
-	dir = 4
+/obj/machinery/atmospherics/components/unary/vent_pump/on/layer2{
+	dir = 1
 	},
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+/obj/machinery/airalarm/directional/south,
+/turf/open/floor/plasteel/dark,
+/area/ship/bridge)
+"wM" = (
+/obj/effect/turf_decal/corner/transparent/bar{
 	dir = 6
 	},
-/turf/open/floor/engine,
-/area/ship/engineering)
-"zG" = (
-/obj/effect/turf_decal/industrial/warning{
+/obj/effect/turf_decal/corner/opaque/syndiered/border{
 	dir = 8
 	},
-/turf/open/floor/plasteel/dark,
-/area/ship/engineering)
-"zJ" = (
-/obj/machinery/atmospherics/pipe/simple/green/visible{
-	dir = 5
-	},
-/turf/closed/wall/r_wall/syndicate/nodiagonal{
-	rad_insulation = 0.1
+/obj/machinery/door/airlock/public/glass,
+/obj/machinery/door/firedoor/border_only{
+	dir = 8
 	},
-/area/ship/engineering/engine)
-"zV" = (
-/obj/effect/turf_decal/industrial/warning,
-/turf/open/floor/pod/dark,
-/area/ship/cargo)
-"zY" = (
-/obj/structure/bed/roller,
-/obj/machinery/iv_drip,
-/obj/effect/turf_decal/corner/opaque/red/diagonal,
-/turf/open/floor/plasteel/white,
-/area/ship/medical)
-"Ac" = (
-/obj/effect/turf_decal/industrial/warning{
+/obj/machinery/door/firedoor/border_only{
 	dir = 4
 	},
-/turf/open/floor/plating,
-/area/ship/engineering/atmospherics)
-"Ae" = (
-/obj/effect/turf_decal/arrows/red{
-	color = "red";
-	icon_state = "arrows"
-	},
-/turf/open/floor/plasteel/mono/dark,
+/turf/open/floor/plasteel,
 /area/ship/crew/dorm)
-"Af" = (
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
-	dir = 10
+"wP" = (
+/obj/effect/turf_decal/corner/opaque/orange/three_quarters{
+	dir = 8
 	},
-/obj/machinery/airalarm/directional/east,
-/obj/effect/turf_decal/corner/opaque/red/diagonal,
-/turf/open/floor/plasteel/white,
-/area/ship/medical/surgery)
-"Aq" = (
-/obj/effect/turf_decal/corner/opaque/white{
-	dir = 1
+/obj/machinery/power/smes/engineering,
+/obj/structure/cable/yellow{
+	icon_state = "0-8"
 	},
-/obj/effect/turf_decal/corner/opaque/white{
-	dir = 2
+/obj/effect/decal/cleanable/dirt/dust,
+/turf/open/floor/plasteel/dark,
+/area/ship/engineering)
+"wR" = (
+/obj/item/gun/energy/ionrifle,
+/obj/item/gun/ballistic/rocketlauncher/unrestricted{
+	spawnwithmagazine = 0
```

Comment:
REMOVE THIS! Spawnwithmagazine removes the magazine of the weapon itself instead of the ammo, which results in the PML 9 spawning without an internal magazine and being unable to be loaded.